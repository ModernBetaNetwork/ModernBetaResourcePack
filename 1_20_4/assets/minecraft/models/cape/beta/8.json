{
	"parent": "minecraft:cape/cape_8",
	"textures": {
		"0": "item/capes/beta",
		"particle": "item/capes/beta"
	}
}