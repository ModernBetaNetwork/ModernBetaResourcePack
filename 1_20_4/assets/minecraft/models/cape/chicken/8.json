{
	"parent": "minecraft:cape/cape_8",
	"textures": {
		"0": "item/capes/chicken",
		"particle": "item/capes/chicken"
	}
}