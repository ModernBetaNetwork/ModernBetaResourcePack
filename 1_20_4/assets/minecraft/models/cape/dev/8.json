{
	"parent": "minecraft:cape/cape_8",
	"textures": {
		"0": "item/capes/dev",
		"particle": "item/capes/dev"
	}
}