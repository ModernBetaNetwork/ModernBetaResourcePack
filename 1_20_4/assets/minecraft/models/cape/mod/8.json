{
	"parent": "minecraft:cape/cape_8",
	"textures": {
		"0": "item/capes/mod",
		"particle": "item/capes/mod"
	}
}