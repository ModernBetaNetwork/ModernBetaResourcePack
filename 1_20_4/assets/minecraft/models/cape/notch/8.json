{
	"parent": "minecraft:cape/cape_8",
	"textures": {
		"0": "item/capes/notch",
		"particle": "item/capes/notch"
	}
}