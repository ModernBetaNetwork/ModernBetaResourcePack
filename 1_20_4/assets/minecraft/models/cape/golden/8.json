{
	"parent": "minecraft:cape/cape_8",
	"textures": {
		"0": "item/capes/golden",
		"particle": "item/capes/golden"
	}
}